src/main/java/com/assignments/questions/Cylinder.java

```diff
@@ -0,0 +1,24 @@
+package com.assignments.questions;
+
+import java.util.Scanner;
+public class Cylinder {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("enter radius, height");
+        // input Radius
+        int radius = sc.nextInt();
```

Comment:
The radius could have been of type double, although it wasn't stated in the question